plugins/woocommerce/client/legacy/js/admin/wc-shipping-zone-methods.js

```diff
@@ -457,12 +499,12 @@
 					return htmlContent.prop( 'outerHTML' );
 				},
 				addCurrencySymbol: function( html ) {
-					if ( ! window.wc.ShippingCurrencyContext || ! window.wc.ShippingCurrencyNumberFormat ) {
+					if ( ! window.wc.wcSettings.CURRENCY || ! window.wc.currency.localiseMonetaryValue ) {
```

Comment:
@rjchow, optional chaining here?
```suggestion
					if ( ! window.wc.wcSettings.CURRENCY || ! window.wc.currency?.localiseMonetaryValue ) {
```

Reply:
I initially started off with it but then I realised we don't transpile this file so it doesn't get polyfilled for lower envs. But I just looked up the browser support and I think it's fine so I'll make the change

![alt text](image.png)